VAMobile/src/store/slices/authSlice.ts

```diff
@@ -699,6 +700,7 @@ export const handleTokenCallbackUrl =
       await logAnalyticsEvent(Events.vama_login_success(SISEnabled))
       await dispatch(dispatchSetAnalyticsLogin())
       dispatch(dispatchFinishAuthLogin({ authCredentials }))
+      usePostLoggedIn()
```

Comment:
Since this isn't in a component, we shouldn't be using a hook here, we can directly call the `postLoggedIn` function. Also, since `postLoggedIn` is only used here, we probably don't need to store it in a separate file. It doesn't seem like something we'll need for other parts of the app

Reply:
updated